### PR TITLE
Made readme accurate to current build process

### PR DESCRIPTION
**Pull request by int-72h**
I added pretty much everything I think.

### CODE REVIEW

**Review by HurricanePootis — APPROVED** [review]
Pog

File: README.md [hunk 1]
```diff
@@ -85,25 +85,33 @@ Other launch options to consider:
...
 
 Instructions:
-1. Prepare for unforseen consequences
-1. Run `./magic.sh -f`
+
+0) Make sure the path leading up the `team-comtress-2` has no spaces in it. This will lead to errors compiling protobud and gperf.
+1) Prepare for unforseen consequences (mainly it crashing during compilation.)
+2) Run `./build.sh`
+2.5) To make sure it compiled correctly, run it again. If you see no errors, then it compiled. Success!
+3) Install your TF2 assets into `../game/tf`, the `game` folder should be outside the repo folder (Note: the `../game/tf` folder will already be there as it's made from the compilation). If you have your TF2 installation on the same drive and location that you have Steam installed on, running `cp -rn '~/.steam/steam/steamapps/common/Team Fortress 2/*' ./` will work.
```
Comment (melvyn2) [inline README.md:112]: Command should be `cp -rn ~/.steam/steam/steamapps/common/Team\ Fortress\ 2/* ./`.

This so also a bit wordy, could this be simplified?

File: README.md [hunk 2]
```diff
@@ -85,25 +85,33 @@ Other launch options to consider:
...
+- libunwind
 - glibc
 - freetype
 - fontconfig
 - libGL
 - libX11
 - openal
 - libncurses
-- curl with GNUTLS support
+- curl with GNUTLS support (note: if you get an error in compiling due to `-lcurl-gnutls` being missing, you need a symlink: `sudo ln -s /usr/lib32/libcurl-gnutls.so.4 /usr/lib32/libcurl-gnutls.so` *Note: only tested on Arch based distros.*)
```
Comment (melvyn2) [inline README.md:104]: should be `libcurl-gnutls`. Also, remove the first 'note:'.

File: README.md [hunk 3]
```diff
@@ -85,25 +85,33 @@ Other launch options to consider:
...
 Instructions:
-1. Prepare for unforseen consequences
-1. Run `./magic.sh -f`
+
+0) Make sure the path leading up the `team-comtress-2` has no spaces in it. This will lead to errors compiling protobud and gperf.
+1) Prepare for unforseen consequences (mainly it crashing during compilation.)
+2) Run `./build.sh`
+2.5) To make sure it compiled correctly, run it again. If you see no errors, then it compiled. Success!
+3) Install your TF2 assets into `../game/tf`, the `game` folder should be outside the repo folder (Note: the `../game/tf` folder will already be there as it's made from the compilation). If you have your TF2 installation on the same drive and location that you have Steam installed on, running `cp -rn '~/.steam/steam/steamapps/common/Team Fortress 2/*' ./` will work.
+4) Run in the `../game` folder `./run.sh`
```
Comment (melvyn2) [inline README.md:113]: `run.sh` is in the source root folder, not game.

**Review by melvyn2 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (int-72h) [reply README.md:104]: A lot of people got an error that was specifically that. I don't see why I need to remove it.

File: README.md (hunk 1 above)
Comment (int-72h) [reply README.md:112]: The commands do the same thing.
Will try and simplify wording.

File: README.md (hunk 2 above)
Comment (melvyn2) [reply README.md:104]: I misspoke: I meant remove the word 'note:', and keep the rest.

Comment (int-72h) [reply README.md]: Done.

File: README.md [hunk 4]
```diff
@@ -85,25 +85,36 @@ Other launch options to consider:
...
+- libcurl-gnutls 
+
+Note: if you get an error in compiling due to `-lcurl-gnutls` being missing, you need a symlink: `sudo ln -s /usr/lib32/libcurl-gnutls.so.4 /usr/lib32/libcurl-gnutls.so`
+
 
 Instructions:
-1. Prepare for unforseen consequences
-1. Run `./magic.sh -f`
+
+0) Make sure the path leading up the `team-comtress-2` has no spaces in it. This will lead to errors compiling protobud and gperf.
```
Comment (melvyn2) [inline README.md:111]: Typo (protobud)

File: README.md [hunk 5]
```diff
@@ -85,25 +85,36 @@ Other launch options to consider:
...
+1) Prepare for unforseen consequences (mainly it crashing during compilation.)
+2) Run `./build.sh`
+2.5) To make sure it compiled correctly, run it again. If you see no errors, then it compiled. Success!
+3) Install your TF2 assets into `../game/tf`. If you have TF2 and steam installed on the same drive, running `cp -rn '~/.steam/steam/steamapps/common/Team Fortress 2/*' ./` will work.
```
Comment (melvyn2) [inline README.md:115]: Still wrong command.

**Review by melvyn2 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (melvyn2) [reply README.md]: no, the single quotes cause the glob not to be expanded.

File: README.md (hunk 4 above)
Comment (int-72h) [reply README.md:111]: oof.

File: README.md (hunk 5 above)
Comment (int-72h) [reply README.md:115]: the command does the same thing. I'll change it though.

Comment (int-72h) [reply README.md:115]: wait- saw your other comment sorry

File: README.md (hunk 1 above)
Comment (int-72h) [reply README.md]: Really? this was pasted from the discord straight and it worked for me. I'll change it anyway.

**Comment by int-72h** [comment]
ok weird, it's deleted the shaderdx8 file

**Comment by int-72h** [comment]
fixed

**Comment by int-72h** [comment]
@melvyn2 why is this closed?

**Comment by int-72h** [comment]
Oh wait- I'll remake this pr on master

**Comment by melvyn2** [comment]
@int-72h whoops, what?
Didn't mean to close this

**Comment by melvyn2** [comment]
Oh that was because I deleted the branch.

**Comment by int-72h** [comment]
I'll remake it on the master branch
